### PR TITLE
Tidy Up

### PR DESCRIPTION
**Pull request by markmur**
- Added custom debug logs to recognise hook logs amongst the others
- Removing jquery and unnecessary scripts for logged in users
- Added firstName and lastName values to inputs if register was unsuccessful
- Email input changed from input[type="text"] to input[type="email"]
- Removed unused model attributes

### CODE REVIEW

File: api/models/User.js [hunk 1]
```diff
@@ -3,50 +3,20 @@ var promise       = require('promised-io/promise');
 var adBox         = require('boxfish-router');
 var colors        = require('colors');
 
+var customLogger = function customLogger() {
```
Comment (danieldmo) [inline api/models/User.js:6]: I feel like this should be in another file ( maybe another npm module ? ) so ir can be used in all files across the hook

Comment (markmur) [inline api/models/User.js:6]: I guess but do you reckon it's big enough to be its own module? I guess we
could use it as a helpers module

Comment (danieldmo) [inline api/models/User.js:6]: if it is going to be used in all hooks it should be a module, a tiny one

File: views/layout.ejs [hunk 2]
```diff
@@ -48,88 +48,22 @@
...
-
-      <% if (req.session.user) { %>
-        // Set the user context for raven error logging
-        Raven.setUserContext({
-          email: '<%= req.session.user.email %>',
-          id: '<%= req.session.user.id %>'
-        });
+        <script>
+          // Sentry configuration
+          Raven.config('https://57ddbec2382c4d02b3fc420a82fc3fd8@app.getsentry.com/59627').install();
```
Comment (danieldmo) [inline views/layout.ejs]: I think we can remove this, this is for Sentry and we no longer use it

Comment (markmur) [inline views/layout.ejs]: Cool yeah I wasn't sure so I left it

File: api/models/User.js (hunk 1 above)
Comment (markmur) [inline api/models/User.js:6]: @danieldmo want to merge this for now anyway? I'll create a jira for a logger repo

**Pull request merged**